src/sentry_plugins/splunk/plugin.py

```diff
@@ -303,13 +243,4 @@ def forward_event(self, event, payload):
                 },
             )
 
-            if isinstance(exc, (ConnectTimeout, ReadTimeout)):
```

Comment:
@NisanthanNanthakumar I think this code is useful so we don't generate more Sentry errors. What I was saying was that we should just update the errors we check for here (`ConnectTimeout` -> `ApiHostError`). See what errors we are raising here:
https://github.com/getsentry/sentry/blob/master/src/sentry/shared_integrations/client.py#L249-L254